### ISSUE TITLE
Evaluate() return value

### ISSUE THREAD

**Issue by 2kaud** [m-peko/booleval#35]
Wouldn't it be better if evaluate() returns false if is_activated_ is false? Returning true gives the wrong impression that the expression has been evaluated when it hasn't if is_activated_ is false??

**Comment by m-peko** [comment]
@2kaud Yes, I agree with you. If you prefer, you can create a PR. Otherwise, I will make this update ASAP.
